### PR TITLE
Add gas giant controversy posters

### PR DESCRIPTION
**Pull request by coiax**
:cl: ArcaneMusic, Twaticus, coiax
add: Nanotrasen reminds its crew members that the station has never
orbited a Gas Giant, and is engaging in a media campaign to remind
employees of this fact.
add: WAke uP ShEepLe
naNotrAsen is LyINg To You
wHERE DoEs THE gaS CoME from, huh?
/:cl:

There is a half remembered fact that Space Station 13 orbits a gas
giant, allegedly due to some entry on a wiki someplace.

Given the vigorous discussion on this subject, it seems appropriate that
this confusion is forever immortalised in poster form.

With thanks to @ArcaneMusic, and @Twaticus  for doing the sprites.

---

image

Some stylish modeling by Braxton Cherry next to the poster.

---

But what's this? A contraband poster in maintenance disputing Nanotrasen's claims?

image

### CODE REVIEW

File: code/game/objects/effects/contraband.dm [hunk 1]
```diff
@@ -615,4 +615,10 @@
 	desc = "A poster advertising the escapades of Dick Gumshue, mouse detective. Encouraging crew to bring the might of justice down upon wire saboteurs."
 	icon_state = "poster36_legit"
 
+/obj/structure/sign/poster/official/there_is_no_gas_giant
+	name = "There Is No Gas Giant"
+	desc = "Nanotrasen has issued posters, like this one, to all stations reminding them that rumours of a gas giant are false."
```
Comment (ATH1909) [inline code/game/objects/effects/contraband.dm:620]: surely at least one of their stations must orbit a gas giant?

and the way this is phrased, it sounds like they're trying to say that gas giants just don't exist anywhere

**Comment by Angustmeta** [comment]
Found 'em

File: code/game/objects/effects/contraband.dm (hunk 1 above)
Comment (Fikou) [reply code/game/objects/effects/contraband.dm:620]: GAS GIANTS DONT EXIST!!

**Comment by Qustinnus** [comment]
kor and oranges killed the gas giant myth in 2017

**Comment by tralezab** [comment]
https://github.com/coiax/tgstation/pull/8

**Comment by cacogen** [comment]
@Angustmeta Probably copied over from when Errorage hosted the wiki.

File: code/game/objects/effects/contraband.dm (hunk 1 above)
Comment (cacogen) [inline code/game/objects/effects/contraband.dm:620]:
```suggestion
	desc = "Nanotrasen has issued posters, like this one, to all stations reminding them that rumours of any of their stations orbiting a gas giant are false."
```

Comment (Dorsisdwarf) [reply code/game/objects/effects/contraband.dm:625]: Nanotrasen is quite firm that gas giants do not exist. Ridiculous.

**Comment by Rohesie** [comment]
I remember hearing about line breaks in the changelog not looking good. Could be wrong.

**Review by cacogen — APPROVED** [review]
I don't know how to close this review UI I accidentally opened

File: code/game/objects/effects/contraband.dm (hunk 1 above)
Comment (Zytolg) [reply code/game/objects/effects/contraband.dm:625]: Gas Planet Canon
Gas Planet Real
Don't believe in Syndicate Lies

**Review by Mothblocks — APPROVED** [review]
Code ✔

**Comment by Twaticus** [comment]
@coiax looks good but it wasnt me who sprited it lol. ill merge after the credit is fixed

**Comment by Rohesie** [comment]
Do fix the changelog first, please!

**Comment by Jared-Fogle** [comment]
> I remember hearing about line breaks in the changelog not looking good. Could be wrong.

He's been doing this for a while, it apparently works alright. @Rohesie

**Comment by Rohesie** [comment]
> 
> 
> > I remember hearing about line breaks in the changelog not looking good. Could be wrong.
> 
> He's been doing this for a while, it apparently works alright. @Rohesie

aha, alrighty, disregard then

**Pull request merged**